crates/forge_analyzer/src/definitions.rs

```diff
@@ -2981,11 +3503,42 @@ impl Environment {
             DefKind::Undefined => DefKind::Undefined,
         }
     }
+    pub fn is_import(&self, import_kind: ImportKind, named_import: Option<String>) -> bool {
```

Comment:
This should probably just be a method an `ImportKind`, since we don't use the resolver here. Also, this could probably just take in a `&str` and check if the import is the same identifier as what was passed in.